### DISCUSSION TITLE
BandMap ideas?

### DISCUSSION THREAD

**Discussion by awandahl** [mbridak/not1mm#226] [Ideas]
The BandMap is very important and different people may have different preferences. A highly configurable BandMap is probably a good idea but being so it will introduce complexity in configuration, use and not the least in programming.

The colors of the spots/marked callsigns can be very useful indicators of the status of a callsign in the BandMap: cluster spot or marked station?, is the station already worked?, is the call a multiplier (dxcc) needed on the present band?, zone (and dxcc) needed on present band?...

This color scheme can also be *consistent* throughout the program and the contest, and can be reflected also in the callsign window, e.g. a potential dupe (already worked station) can be red, a needed multiplier (dxcc) can be green, a needed zone can be... This will make it easier to prioritize well when tuning the band in S&P mode.

**Comment by sp3rat** [comment]
I'm just starting to work with not1mm.
It's inconvenient for me that the ssb on 40m VFO is set to 7.250MHz
and on 80m VFO is set to 3.970MHz

**Comment by py4xx** [comment]
Hi Mike...
Hi Anders...

I also think that a cluster filter is very important in BandMaper, so that when we want spots from just one band, we can do this with just one click. 73

![image](https://github.com/user-attachments/assets/a76ae85c-beff-4202-9ffe-51b3b61e2a3c)
